stores/searchResultsStore.ts

```diff
@@ -0,0 +1,153 @@
+import { defineStore } from 'pinia'
+import { Ref, ref } from 'vue'
+
+type searchResult = {
+    id: string;
+    type: string;
+    names: {
+        lastName: string;
+        firstName: string;
+        middleName: string;
+        locale: string;
+    }[];
+    location: string;
+    degrees: string[];
+    spokenLanguages: string[];
+    specialties: specialty[];
+    acceptedInsuranceOptions: string[];
+}
+
+type specialty = {
+    id: string;
+    names: name[];
+}
+
+type name = {
+    name: string;
+    locale: string;
+}
+
+const fakeSearchResultsList : searchResult[] = [
```

Comment:
future idea: We could move this to a fakedata folder to be used in tests. For now, this is fine since we haven't hooked up the API yet.

Reply:
That's a bet.